### PR TITLE
remove redis-slave from test, as this is not required for test success. Upgrade redis chart

### PR DESCRIPTION
**Pull request by OlegLoewen**
**What this PR does / why we need it**:
remove redis-slave from test, as this is not required for test success. Upgrade redis chart

**Release note**:

```
remove redis-slave from test, as this is not required for test success. Upgrade redis chart
```

### CODE REVIEW

**Comment by OlegLoewen** [comment]
@schrodit @dguendisch could you also take a look?

File: go.mod [hunk 1]
```diff
@@ -3,9 +3,13 @@ module github.com/gardener/gardener
 go 1.13
 
 require (
+	contrib.go.opencensus.io/exporter/ocagent v0.4.12 // indirect
```
Comment (ialidzhikov) [inline go.mod]: Can you revert the `go.mod` changes. Probably you are using golang@1.12? If yes, can you update to golang@1.13?

Comment (OlegLoewen) [reply go.mod]: thx, done

File: test/integration/framework/applications/guestbooktest.go [hunk 2]
```diff
@@ -110,25 +105,22 @@ func (t *GuestBookTest) DeployGuestBookApp(ctx context.Context, gardenerTestOper
 	gomega.Expect(err).NotTo(gomega.HaveOccurred())
 
 	ginkgo.By("Applying redis chart")
+	// redis-slaves are not required for test success
+	chartOverrides := map[string]interface{}{
+		"cluster": map[string]interface{}{
+			"enabled": false,
+			"slaveCount": 0,
```
Comment (ialidzhikov) [inline test/integration/framework/applications/guestbooktest.go]: Can we skip specifying this as `slaveCount ` is the replicas of the slave sts, but we have `enabled: false` which means that no slave sts is created.
```suggestion
			"slaveCount": 0,
```

File: test/integration/framework/applications/guestbooktest.go [hunk 3]
```diff
@@ -44,13 +44,11 @@ const (
 	// RedisMaster is the name of the redis master deployed by the helm chart
 	RedisMaster = "redis-master"
 
-	// RedisSlave is the name of the redis-slave deployed by the helm chart
-	RedisSlave = "redis-slave"
```
Comment (ialidzhikov) [inline test/integration/framework/applications/guestbooktest.go]:
```suggestion

```

File: test/integration/framework/applications/guestbooktest.go (hunk 2 above)
Comment (OlegLoewen) [reply test/integration/framework/applications/guestbooktest.go]: done

**Review by rfranzke — APPROVED** [review]

**Review by ialidzhikov — APPROVED** [review]
/approve

**Comment by rfranzke** [comment]
Sorry, I missed that additional reviews were pending.

**Review by dguendisch — COMMENTED** [review]
Look_ed_-good-to-me :)

**Pull request merged**